### PR TITLE
added StatutePage

### PR DESCRIPTION
**Pull request by Shembou**

### CODE REVIEW

File: src/app/(main)/regulamin/page.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import { Header_Query } from '@/components/_legal/Header';
+import { Contents_Query } from '@/components/_legal/Contents';
+import Seo, { Seo_Query } from '@/global/Seo';
+
+const page = { name: 'Regulamin', path: '/regulamin' };
+
+export default async function StatutePage() {
+  const {
+    global: { tel, email },
+    statute_page: { Content, Header_Description, Header_Heading, Files },
```
Comment (milewskibogumil) [inline src/app/(main)/regulamin/page.tsx:17]: change `statute_page` to simply `page`

File: src/app/(main)/regulamin/page.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { draftMode } from 'next/headers';
+import Breadcrumbs from '@/components/_global/Breadcrumbs';
+import Contents from '@/components/_legal/Contents';
+import FilesComponent, { FilesComponent_Query } from '@/components/_legal/FilesComponent';
+import Header from '@/components/_legal/Header';
+import { StatutePageQueryProps } from '@/global/types';
```
Comment (milewskibogumil) [inline src/app/(main)/regulamin/page.tsx:6]: always add `type` before import of types in TS.
Example:
```
import type { StatutePageQueryProps } from '@/global/types';
```

Comment (milewskibogumil) [reply src/app/(main)/regulamin/page.tsx:6]: And move types for the end of the import lines

File: src/app/(main)/regulamin/page.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        tel,
+        email
+      },
+      'statute_page': *[_id=='Statute_Page'][0] {
+        ${Header_Query}
+        ${Contents_Query}
+        ${FilesComponent_Query}
+        ${Seo_Query}
+      }
+    }
```
Comment (milewskibogumil) [inline src/app/(main)/regulamin/page.tsx:57]: One more TAB is missing here

File: src/components/_legal/Contents/Contents.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+'use server';
```
Comment (milewskibogumil) [inline src/components/_legal/Contents/Contents.tsx:1]: `use server` is unnecessary

File: src/components/_legal/FilesComponent/FilesComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+const FilesComponent = ({ data }: Props) => {
+  return (
+    <section className={styles['FilesComponent']}>
+      {data.map(({ asset: { originalFilename, size, url } }, index) => (
+        <div
+          className={styles.linkWrapper}
+          key={index}
+        >
+          <a
+            href={`${url}?dl=${originalFilename}`}
```
Comment (milewskibogumil) [inline src/components/_legal/FilesComponent/FilesComponent.tsx:14]: What is the `?dl` used for?

File: src/components/_legal/FilesComponent/FilesComponent.types.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+export type Props = {
+  data: Array<{
+    asset: {
+      url: string;
+      originalFilename: string;
+      size: number;
+    }
+  }>;
```
Comment (milewskibogumil) [inline src/components/_legal/FilesComponent/FilesComponent.types.ts:8]: Use one convention. Above you've use `{}[]`, and now you're using `Array<{}>`.
Let's make an agreement to always use `{}[]` in TypeScript to mark arrays.

File: src/components/_legal/Header/Header.module.scss [hunk 7]
```diff
@@ -0,0 +1,35 @@
+.Header {
+  display: grid;
+  justify-content: center;
+  max-width: 690px;
```
Comment (milewskibogumil) [inline src/components/_legal/Header/Header.module.scss:4]: `max-width` should always be declared as `rem`. In that case `calc(690rem/16)`

**Review by milewskibogumil — CHANGES_REQUESTED** [review]

File: src/components/_legal/FilesComponent/FilesComponent.tsx (hunk 5 above)
Comment (Shembou) [reply src/components/_legal/FilesComponent/FilesComponent.tsx:14]: It's a sanity feature that lets you download a file without leaving the site: https://www.sanity.io/docs/file-type

**Pull request merged**